demo/src/pick_place_task.cpp

```diff
@@ -82,20 +82,22 @@ void PickPlaceTask::loadParameters() {
 	rosparam_shortcuts::shutdownIfError(LOGNAME, errors);
 }
 
+// The init function declares all movements for the pick-and-place task. These movements are described below.
 void PickPlaceTask::init() {
 	ROS_INFO_NAMED(LOGNAME, "Initializing task pipeline");
 	const std::string object = object_name_;
 
 	// Reset ROS introspection before constructing the new object
 	// TODO(henningkayser): verify this is a bug, fix if possible
+	// Movements are collected inside a Task object.
```

Comment:
A bit confusing with the previous two lines. Make this a line-end comment (put it at the end of the next line with two separating spaces).